vhdbuilder/packer/install-dependencies.sh

```diff
@@ -136,12 +136,20 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
   installStandaloneContainerd ${containerd_version} ${containerd_patch_version}
   echo "  - [installed] containerd v${containerd_version}-${containerd_patch_version}" >> ${VHD_LOGS_FILEPATH}
 
-  CRICTL_VERSIONS="
-  1.22.0
-  1.23.0
-  1.24.0
-  1.25.0
-  "
+  DOWNLOAD_FILES=$(jq ".DownloadFiles" $COMPONENTS_FILEPATH | jq .[] --monochrome-output --compact-output)
+  for componentToDownload in ${DOWNLOAD_FILES[*]}; do
+    fileName=$(echo "${componentToDownload}" | jq .fileName -r)
+    if [ $fileName == "crictl-v*-linux-amd64.tar.gz" ]; then
+      CRICTL_VERSIONS_STR=$(echo "${componentToDownload}" | jq .versions -r)
+      CRICTL_VERSIONS=""
+      if [[ ${CRICTL_VERSIONS_STR} != null ]]; then
+        CRICTL_VERSIONS=$(echo "${CRICTL_VERSIONS_STR}" | jq -r ".[]")
+      fi
+      break
+    fi
+  done
+  echo $CRICTL_VERSIONS
+
   for CRICTL_VERSION in ${CRICTL_VERSIONS}; do
     downloadCrictl ${CRICTL_VERSION}
```

Comment:
dowwnloadcrictl can also be refactored to use the download URL present in components.json. Since the URL is not expected to change soon (and we need to include parts like CPU arch), that refactoring can be done in a future PR.